config/dbConn.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require('mongoose');
+
+const connectDB = async () => {
+  try {
+    await mongoose.connect(
+      process.env.DATABASE_URI, 
+    );
+  } catch (err) {
+    console.error(err)
+  }
+}
+
+module.exports = connectDB;
```

Comment:
This is how you connect to MongoDB- it's like a cloud instance of a DB. Mongoose is the package that acts like a wrapper/ORM/interface with MongoDB for Node.js